src/nhsd/components/molecules/filter-menu-section/filter-menu-section.js

```diff
@@ -0,0 +1,27 @@
+export default class NHSDFilterMenu {
+  constructor(componentEl) {
+    if (!componentEl) return null;
+    this.hostEl = componentEl;
+    this.init();
+  }
+
+  /* eslint-disable */
+  init() {
+    let i;
+
+    for (i = 0; i < this.hostEl.children.length; i++) {
```

Comment:
`i` should be initialised in for loop, `for (let i = 0; i < this.hostEl.children.length; i++)`